tests/Integration/Session/DatabaseSessionHandlerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace Illuminate\Tests\Integration\Session;
+
+use Illuminate\Session\DatabaseSessionHandler;
+use Illuminate\Support\Carbon;
+use Illuminate\Tests\Integration\Database\DatabaseTestCase;
+
+class DatabaseSessionHandlerTest extends DatabaseTestCase
+{
+    private mixed $migration;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->migration = require __DIR__.'/../../../src/Illuminate/Session/Console/stubs/database.stub';
+        $this->migration->up();
```

Comment:
You don't need this. https://packages.tools/testbench/the-basic/databases.html#run-laravel-migrations `WithMigration` attribute would solve this

Reply:
Good point. Done.